src/components/Create/components/JuiceVideoOrImgPreview.tsx

```diff
@@ -2,6 +2,7 @@ import { CloseOutlined } from '@ant-design/icons'
 import { Image, ImageProps } from 'antd'
```

Comment:
**suggestion** this component is used outside of the Create flow. Should it be moved to the `components/` directory?

Reply:
Done